Misc/NEWS.d/next/Build/2024-04-15-08-35-06.gh-issue-117845.IowzyW.rst

```diff
@@ -0,0 +1 @@
+Fixed function pointer type mismatch when building the readline module against recent libedit versions.
```

Comment:
Thanks for adding a NEWS entry! I think we should try to word it so it is clearer what _problem_ this solves for the user. What do you think of this suggestion?

```suggestion
Fix building against recent libedit versions by detecting readline hook signatures in :program:`configure`.
```

Reply:
Looks fine to me.